doc/source/reference/general_functions.rst

```diff
@@ -61,6 +61,7 @@ Top-level dealing with datetimelike data
 
 Top-level dealing with Interval data
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 .. autosummary::
    :toctree: api/
```

Comment:
extra line, but it doesn't seem to make any difference in the build so is OK imo

![image](https://github.com/pandas-dev/pandas/assets/33491632/d9b301f8-1379-4ce6-8363-573f652f7915)